libr/anal/class.c

```diff
@@ -424,6 +424,13 @@ static void r_anal_class_set_flag(RAnal *anal, const char *name, ut64 addr) {
 	anal->flg_class_set (anal->flb.f, name, addr, 0);
 }
 
+static void r_anal_class_vtable_set_flag(RAnal *anal, const char *name, ut64 addr, ut32 size) {
```

Comment:
The meaning of this function isn't really specific to vtables. You can use it as a direct replacement of `r_anal_class_set_flag()` and just pass 0 as the size at the old callsites.

Reply:
Yes, I a changed back to using r_anal_class_set_flag()